setup.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+from setuptools import setup, find_packages
+
+requirements = [
+    'pulpcore-plugin>=0.1.0b7',
```

Comment:
Where does openssl come from as a requirement?

Reply:
I think it's just pip installable right?

Reply:
Seems like only pip `PyOpenSSL`